pandas/tests/io/pytables/test_round_trip.py

```diff
@@ -357,7 +357,7 @@ def test_timeseries_preepoch(setup_path):
         _check_roundtrip(ts, tm.assert_series_equal, path=setup_path)
     except OverflowError:
         if is_platform_windows():
-            pytest.xfail("known failure on some windows platforms")
+            pytest.mark.xfail("known failure on some windows platforms")
```

Comment:
I think this will make it appears that this test will pass when it hits this line as this mark is not attached to anything.

Probably needs something like

```
except OverflowError:
     if is_platform_windows():
         request.node.add_marker(pytest.mark.xfail(reason=...)
     raise
```

Reply:
right, thanks!